internal/pkg/local/manager.go

```diff
@@ -136,6 +138,26 @@ func (u *UnitOfWork) LoadObject(record model.Record) {
 	u.
 		workersFor(record.Level()).
 		AddWorker(func() error {
+			// Has been parent loaded?
```

Comment:
Pri remote load, ak sa nepodari nacitat `parent`, tak sa nenacitavaju ani `children`.

Nepodari sa nacitat `branch`, nenacitavaju sa ani `configs` (je tam `continue`):
https://github.com/keboola/keboola-as-code/blob/78a1194c138dc7730fd17128c5a0a0774d01bd99/internal/pkg/remote/manager.go#L76-L87

Podobne, ak sa nepodari nacitat `config`, tak ani `rows`:
https://github.com/keboola/keboola-as-code/blob/78a1194c138dc7730fd17128c5a0a0774d01bd99/internal/pkg/remote/manager.go#L101-L121

---------------------------------------------

No a toto chyba pri `local load`.
Vysledok je, ze ak je chyba v `config`, tak sa nenacita.
Ale rows sa nacitali. A potom to spadlo v `shared code` mapper, ... kde potrebujeme config, a je tam `MustGet`.